EIPS/eip-2976.md

```diff
@@ -0,0 +1,90 @@
+---
+eip: 2976
+title: eth/##: Typed Transactions over Gossip
```

Comment:
I believe this needs to be wrapped with double quotes to render properly on the eips site. This is because the front matter parser thinks "eth/##:" is a key like "title:". This is on the backlog to validate with eipv.

Reply:
Will do.  I copied the name from the other networking EIPs, looking again it does appear they have quotes.